discord_slash/dpy_overrides.py

```diff
@@ -11,6 +13,20 @@ def __init__(self, *, state, channel, data):
         super().__init__(state=state, channel=channel, data=data)
         self.components = data["components"]
 
+    def get_component(self, custom_id: int) -> typing.Optional[dict]:
+        """
+        Returns first component with matching custom_id
+
+        :param custom_id: custom_id of component to get from message components
+        :return: Optional[dict]
+
+        """
+        for row in self.components:
+            for component in row["components"]:
+                if component["custom_id"] == custom_id:
```

Comment:
`is` operator would be a faster method to approaching checking the component's custom ID, especially if we have numerous ones and many action rows, likely a better approach IMO for large-scaled commands in bots.